tmail-backend/apps/distributed/pom.xml

```diff
@@ -93,6 +93,11 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>tmail-mailets</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>tmail-openpaas</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```

Comment:
Move the version declaration into `tmail-backend/pom.xml`, use `${project.version}` there and here just declare the dependency without the version.